website/docs/guides/providers.md

```diff
@@ -0,0 +1,43 @@
+---
+id: provider
+title: Manage providers
+sidebar_position: 1
+---
+
+### Before you start
+
+You can manage a provider assigned to a specific project.
+Firstly, you need to be registered to the accounting-system and then you need to contact the administrator of the project, that this provider is associated with, to assign you one or more roles on the provider. 
+
+**ΝΟΤΕ** In accounting sytem, already exists the **_provider_admin_** role , that permits the user to perform any operation, on a specific provider. In case the user is assigned with any other role, he can operate according to the role's permissions.
+
+## OPERATIONS 
+
+--- 
+
+### READ the provider
+<details>
+You can display the providers's details. Apply a request to the api. 
+<b> For more details,how to syntax the request, see <a href="https://argoeu.github.io/argo-accounting/docs/api/provider#get---fetch-an-existing-provider">here</a></b>
+</details>
+
+### ASSIGN installations to the provider
+<details>
+You can assign one or more installations to the provider. Apply a request to the api.
+<b> For more details,how to syntax the request, see <a href="https://argoeu.github.io/argo-accounting/docs/api/installation#post---create-a-new-installation">here</a></b>
+</details>
+
+### COLLECT all installations assigned to the provider
+<details>
+You can collect and display the details of all the installations assigned to the provider. Apply a request to the api
+<b> For more details,how to syntax the request, see <a href="https://argoeu.github.io/argo-accounting/docs/api/installation#get-fetch-all-provider-installations">here</a></b>
+</details>
+
+### SEARCH installations
+<details>
+You can search for specific installation assigned to the provider, that matches one or more criteria. Apply a request to the api. You need to provide the search criteria in a specific <b><a href="https://argoeu.github.io/argo-accounting/docs/guides/search-filter"> syntax</a></b>. <b> For more details,how to syntax the request, see <a href="https://argoeu.github.io/argo-accounting/docs/api/installation#post---search-for-installations">here</a></b>
```

Comment:
Could you please add specific criteria of the search. 
Not how to .. but what i can search